### ISSUE TITLE
Feature: support Taskfile aliases

### ISSUE THREAD

**Issue by tennox** [linux-china/task-keeper#12]
[Taskfile Docs](https://taskfile.dev/usage/#task-aliases)

Example `Taskfile.yml`:

```yaml
version: '3'
tasks:
  generate:
    aliases: [gen]
    cmds:
      - task: gen-mocks
```

### Current behaviour
- alias is not found
- list shows cut-off description

```
❯ task --list-all
task: Available tasks for this project:
* generate:             (aliases: gen)
❯ tk gen
[tk] no tasks found
❯ tk -l
Available task runners:
  task: Taskfile.yml - https://taskfile.dev
    -- default
    -- generate : (aliases
```

**Comment by tennox** [comment]
Task names with colons in them are also broken:

```
❯ task --list-all
task: Available tasks for this project:
* db:live-dump:
* db:pull:

❯ tk -l
Available task runners:
  task: Taskfile.yml - https://taskfile.dev
    -- db : live-dump
    -- db : pull
❯ tk db:pull
[tk] no tasks found
```

Seems to parse the part after the colon as description

**Comment by linux-china** [comment]
Fixed, please try version 0.24.0
